site/src/content/docs/roadmap.mdx

```diff
@@ -1,86 +1,75 @@
-- [ ] - Gather and prepare to meet OpenSSF's `incubation` requirements.
-- [ ] - Officially cut a GA v1.0.0 release of Zarf.
+---
 
-## Feature Stability
+## 2025 Development Story
 
-Zarf has three levels of feature stability that are defined below but generally follow existing Kubernetes conventions. These communicate how a feature may change in the future as well as expectations for [\*ilities](https://en.wiktionary.org/wiki/ility) like documentation.
+### SDK Readiness (Q1 - Q2)
 
-### `alpha`
+We began the year with a deliberate push to lay the groundwork for SDK support. This involves:
 
-Alpha features act as early previews for things that are being considered for inclusion into Zarf and:
+- Refactoring internal logic to split the CLI from the shared library.
+- Establishing SDK entry points for Zarf functionality.
+- Introducing shared serialization and validation tooling to enable custom tooling built on Zarf.
 
-- they are very likely to have breaking changes (or be removed entirely) in future versions of Zarf
-- they will have limited documentation beyond autogenerated CLI and schema documentation (questions should be directed to the [Zarf team](https://kubernetes.slack.com/archives/C03B6BJAUJ3))
-- they will have limited testing and are only recommended for use cases where the risk of instability is acceptable
+Parallel to these efforts, we also focused on extending our test suite to cover decoupled components and ensure changes did not regress GA stability.
 
-### `beta`
+### Hardening & Observability (Q2 - Q3)
 
-Beta features are newer features that are being prepared to eventually become stable features and:
+As the architecture matured, attention turns to reliability and transparency:
 
-- they may introduce breaking changes before release, but will include any applicable workarounds
-- they will have documentation beyond autogenerated docs, but it won't be fully complete
-- they will have tests that cover their core features, but not covering every flavor of environment
+- Added in-depth error messages and diagnostic metadata to simplify user troubleshooting.
+- Hardened lifecycle to recover from partial failures or restarts.
+- Standardize on logging patterns for package lifecycle execution.
 
-### `stable`
+These upgrades strengthen Zarf's use in production pipelines and across disconnected systems.
 
-Stable features are features that the Zarf team has committed to maintaining stability for and:
+### Package Lifecycle and UX Enhancements (Q3 - Q4)
 
-- they will have backwards compatibility shims introduced in lieu of breaking changes
-- they will have autogenerated docs, tutorials/tutorial mentions, examples, and core docs
-- they will have tests that cover the feature which will run across all tested environments
+Actively investing in enhancements that simplify and expand what Zarf can do:
 
-:::note
+- Simplified authoring of `zarf.yaml` with clearer validation errors and editor integrations.
```

Comment:
I feel like we already have solid validation with a check on the schema, and then dynamic validation for what can't be validated by the schema. What else are you thinking we would add?

Reply:
agreed - It was a bit of a softball to be honest. I'll remove this one.